src/cache-utils.ts

```diff
@@ -111,50 +112,38 @@ export const getPackageManagerInfo = async (packageManager: string) => {
 };
 
 /**
+ * Expands (converts) the string input `cache-dependency-path` to list of directories that
+ * may be project roots
+ *
  * getProjectDirectoriesFromCacheDependencyPath is called twice during `restoreCache`
  *  - first through `getCacheDirectories`
  *  - second from `repoHasYarn3ManagedCache`
  *
- *  it contains expensive IO operation and thus should be memoized
- */
-
-let projectDirectoriesMemoized: string[] | null = null;
-/**
- * unit test must reset memoized variables
- */
-export const resetProjectDirectoriesMemoized = () =>
-  (projectDirectoriesMemoized = null);
-/**
- * Expands (converts) the string input `cache-dependency-path` to list of directories that
- * may be project roots
+ * it contains expensive IO operation and thus should be memoized
+ *
  * @param cacheDependencyPath - either a single string or multiline string with possible glob patterns
  *                              expected to be the result of `core.getInput('cache-dependency-path')`
  * @return list of directories and possible
```

Comment:
Nitpick: it looks like the sentence is not finished 
cc @dsame

Reply:
Yes, i double checked the file and the comment must be just
```
 * @return list of directories 
```